rtf_vis_tool/src/Components/frustum_classes/se3.js

```diff
@@ -20,16 +20,16 @@ class SE3 {
         if (translation.shape[0] !== 3) throw new Error('Invalid Translation Matrix');
         this.rotation = rotation;
         this.translation = translation;
-        this.transform_matrix = nj.identity(4);
+        this.cTw = nj.identity(4);
```

Comment:
we can keep this more generic to represent any class, so I would prefer to call it what it was before

Reply:
Cool. I'll change it back